.gitignore

```diff
@@ -44,4 +44,5 @@ bin/
 # Ballerina
 velocity.log*
 *Ballerina.lock
-/examples/**/Dependencies.toml
+examples/**/Dependencies.toml
+load-tests/**/Dependencies.toml
```

Comment:
newline

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/416/commits/324267555ab3315e1c2aa8f0c486fc993bb74647